R/DataLoadingSaving.R

```diff
@@ -315,28 +296,36 @@ getDbCohortMethodData <- function(connectionDetails,
     substring(label, 1) <- toupper(substring(label, 1, 1))
     if (sampled) {
       preSampleCounts$description <- label
-      metaData$attrition <- rbind(metaData$attrition, preSampleCounts)
-      metaData$attrition <- rbind(metaData$attrition, getCounts(cohorts, "Random sample"))
+      metaData$attrition <-
+        rbind(metaData$attrition, preSampleCounts)
+      metaData$attrition <-
+        rbind(metaData$attrition, getCounts(cohorts, "Random sample"))
     } else {
-      metaData$attrition <- rbind(metaData$attrition, getCounts(cohorts, label))
+      metaData$attrition <-
+        rbind(metaData$attrition, getCounts(cohorts, label))
     }
   } else {
     if (sampled) {
       preSampleCounts$description <- "Original cohorts"
       metaData$attrition <- preSampleCounts
-      metaData$attrition <- rbind(metaData$attrition, getCounts(cohorts, "Random sample"))
+      metaData$attrition <-
+        rbind(metaData$attrition, getCounts(cohorts, "Random sample"))
     } else {
       metaData$attrition <- getCounts(cohorts, "Original cohorts")
     }
   }
   delta <- Sys.time() - start
-  message("Fetching cohorts took ", signif(delta, 3), " ", attr(delta, "units"))
```

Comment:
I'm not quite following the logic for when lines are broken apart, and when they are kept as a single line. If it fits, I prefer to keep lines like these as one line.